umm-spec-lib/resources/json-schemas/granule/umm/v1.4/umm-g-json-schema.json

```diff
@@ -252,89 +252,50 @@
       "required": ["DayNightFlag", "ProductionDateTime"]
     },
     "ArchiveAndDistributionInformationType": {
+      "description": "This set of elements describes a file package or a file that contains other files.  Normally this is either a tar or a zip file.",
+      "anyOf": [{"$ref": "#/definitions/FilePackageType"}, {"$ref": "#/definitions/FileType"}]
```

Comment:
Please submit a PR in [EMFD](https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse) for the schema change as well. Thanks!